ClientApp/src/types/sprint.ts

```diff
@@ -0,0 +1,6 @@
+export type SprintData = {
```

Comment:
Would we want a sprint data type to have a array of tasks assigned to it?

Reply:
That's a good point, I have them adjacent to each other in `sprint-dropdown.component.ts` and I could just as easily lump them into the same type. I can make that change right now.